src/Lykke.Sdk/LykkeStarter.cs

```diff
@@ -22,12 +22,8 @@ public static class LykkeStarter
         public static IWebHostFactory WebHostFactory
         {
             get => _webHostBuilderFactory;
-            set
-            {
-                    //Ignores null
-                    if (value != null)
-                        _webHostBuilderFactory = value;
-            }
+            set => _webHostBuilderFactory =
+                    value ?? throw new ArgumentNullException($"{nameof(WebHostFactory)} can't be null");
```

Comment:
There is the argumentName parameter in the every ArgumentXXXException, better to pass nameof(arg) there. Minor, just FYI.